scripts/dev-db-reset.py

```diff
@@ -31,7 +36,14 @@ def populate():
 
 
 if __name__ == '__main__':
-    parse_args()
+    args = parse_args()
+    if args.keep:
+        from vimma.models import Permission
+        try:
+            Permission.objects.all()
```

Comment:
I've already done this, and changed the filename, to be importable.

I'm using your suggestion for checking if the database is there or not with the Permission -model query.